packages/components/tests/dummy/app/templates/components/modal.hbs

```diff
@@ -0,0 +1,669 @@
+      </p>
+    </dd>
+    <dt>onClose <code>function</code></dt>
+    <dd>
+      <p>
+        Callback function invoked when the modal is closed.
+      </p>
+    </dd>
+    <dt>...attributes</dt>
+    <dd>
+      <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
+    </dd>
+  </dl>
+
+  <h4 class="dummy-h4">Contextual components</h4>
+  <p class="dummy-paragraph" id="contextual-components-modal">The title, the content of the modal dialog, and the
+    actions are passed into the modal as yielded components, using the
+    <code class="dummy-code">Header</code>,
+    <code class="dummy-code">Body</code>,
+    <code class="dummy-code">Footer</code>
+    keys.</p>
+  <dl class="dummy-component-props" aria-labelledby="contextual-components-modal">
+    <dt>&lt;[M].Header&gt; <code>yielded component</code> <strong class="required">required</strong></dt>
+    <dd>
+      <p>It is a container that yields its content as the title of the modal dialog.</p>
+      <p>It accepts
+        <code class="dummy-code">...attributes</code>
+        spreading.</p>
+      <dl class="dummy-component-props">
+        <dt>icon <code>string</code></dt>
+        <dd>
+          <p>Acceptable values: any
+            <a href="https://flight-hashicorp.vercel.app/" target="_blank" rel="noopener noreferrer">
+              Flight</a>
+            icon name.</p>
+        </dd>
+        <dt>tagline <code>string</code></dt>
+        <dd>
+          <p>A string that helps the user maintain context when a modal dialog is open.</p>
+        </dd>
+        <dt>...attributes</dt>
+        <dd>
+          <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
+        </dd>
+      </dl>
+    </dd>
+    <dt>&lt;[M].Body&gt; <code>yielded component</code></dt>
+    <dd>
+      <p>It is an unstyled, generic container that yields as the main content of the modal dialog.</p>
+      <p>It accepts
+        <code class="dummy-code">...attributes</code>
+        spreading.</p>
+    </dd>
+    <dt>&lt;[M].Footer&gt; <code>yielded component</code></dt>
+    <dd>
+      <p>It is a container that yields its content as the footer of the modal dialog.</p>
+      <p>We recommend using it exclusively for actions using the
+        <LinkTo @route="components.button-set"><code class="dummy-code">ButtonSet</code></LinkTo>
+        component. If a tertiary action is presented, it will always be aligned at the end of the row.</p>
+      <p>It accepts
+        <code class="dummy-code">...attributes</code>
+        spreading.</p>
+    </dd>
+  </dl>
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="how-to-use"><a href="#how-to-use" class="dummy-link-section">§</a> How to use</h3>
+
+  <h4 class="dummy-h4">Basic use</h4>
+  <p class="dummy-paragraph">The simplest way to invoke a modal is by doing something like this:</p>
+  {{! prettier-ignore-start }}
+  {{! template-lint-disable no-unbalanced-curlies }}
+  <CodeBlock
```

Comment:
[a11y issue] Since we're running into "scrollable regions must have keyboard access" issues here, recommend trying to resolve it (e.g., control it via CSS or putting in some line breaks in the code sample so that no scrolling is required (at least at default size))

Screenshot: 
![CleanShot 2022-10-31 at 17 16 34](https://user-images.githubusercontent.com/4587451/199120860-dc0ae71a-9e1c-4b45-bdd8-3a74b17be981.png)

Reply:
I went with line breaks for now. We'll have to address this issue more widely though.